src/main/java/org/folio/reader/JPathSyntaxEntityReader.java

```diff
@@ -53,6 +54,7 @@ public JPathSyntaxEntityReader(String json) {
 
   @Override
   protected RuleValue readSimpleValue(Rule rule, DataSource dataSource) {
+    log.info("readSimpleValue:: parameters rule: {}, dataSource: {}", rule, dataSource);
```

Comment:
1) Rule is object. I'm not sure that it has custom toString() implementation to show needed information. 2) For which reason we need to log hash code of data source?

Reply:
@khandramai 
The DataSource- class has been improved by ToString Lombok annotation as well as Translation-class.
So the log content will be readable. 
![image](https://github.com/folio-org/generate-marc-utils/assets/141402754/c12a18e4-d0ad-4994-bc30-d94332c775ba)
![image](https://github.com/folio-org/generate-marc-utils/assets/141402754/5c46bb71-31cd-4d0f-8220-ff24cd2d795d)

Reply:
@DmytroBykov1 , please, explain, for which reasons we need to have data source information in logs.